### PR TITLE
:bug: Change for absolute url on trek's description in api v2

### PR DESCRIPTION
**Pull request by LePetitTim**
https://github.com/GeotrekCE/Geotrek-admin/issues/3321

### CODE REVIEW

File: geotrek/api/v2/serializers.py [hunk 1]
```diff
@@ -618,7 +619,28 @@ def get_published(self, obj):
...
+            imgs = soup.find_all('img')
+            for img in imgs:
+                if img.attrs['src'][0] == '/':
+                    img['src'] = f'{self.context.get("request").build_absolute_uri("/")}{img.attrs["src"]}'
+
         def get_description(self, obj):
+            lang = self.context.get('request').GET.get('language', 'all') if self.context.get('request') else 'all'
+
+            if lang != 'all':
+                data = getattr(obj, '{}_{}'.format('description', lang))
```
Comment (marcantoinedupre) [inline geotrek/api/v2/serializers.py:633]: I don't get how `data` is eventually returned.

File: geotrek/api/v2/serializers.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+from bs4 import BeautifulSoup
```
Comment (marcantoinedupre) [inline geotrek/api/v2/serializers.py:1]: Maybe you forgot to add `bs4` to dependencies.

File: docs/changelog.rst [hunk 3]
```diff
@@ -41,6 +41,7 @@ In preparation for HD Views developments (PR #3298)
 - Check pictogram exist on categories during generation of pdfs
 - Fix ApidaeParsers does not update every time
 - Add fixtures licenses initial install
+- Replace trek's description relative url with absolute url on api v2 (#3321)
```
Comment (marcantoinedupre) [inline docs/changelog.rst:44]: - proposed phrasing: "Replace image's relative URLs with absolute URLs in API v2 trek descriptions"

Comment (marcantoinedupre) [reply docs/changelog.rst:44]: Could you add a link to the issue for the "#3321", thx :pray:

**Review by Chatewgne — APPROVED** [review]

File: geotrek/api/v2/serializers.py (hunk 2 above)
Comment (LePetitTim) [reply geotrek/api/v2/serializers.py:1]: beautifulsoup4==4.11.1 requirements.txt

File: geotrek/api/tests/test_v2.py [hunk 4]
```diff
@@ -341,7 +341,10 @@ def setUpTestData(cls):
             reservation_system=cls.reservation_system,
             practice=cls.practice,
             difficulty=cls.difficulty,
-            accessibility_level=cls.accessibility_level
+            accessibility_level=cls.accessibility_level,
+            description='<p>Description</p>'
+                        '<img src="/media/upload/steep_descent.svg" alt="Descent">'
+                        '<img src="https://rando-millevaches-admin.fr/media/upload/pedestre.svg" alt="" width="1848" height="1848">'
```
Comment (submarcos) [inline geotrek/api/tests/test_v2.py:347]:
```suggestion
                        '<img src="https://rando-millevaches-admin.fr/media/upload/pedestre.svg" alt="" width="1848" height="1848">'
```
don't let real URL in public github commit

**Pull request merged**